ilastik/shell/gui/reportIssueDialog.py

```diff
@@ -0,0 +1,114 @@
+import os
+import platform
+import re
+import webbrowser
+from ilastik import __version__ as ilastik_version
+from ilastik import ilastik_logging
+from ilastik.app import STARTUP_MARKER
+from PyQt5.QtWidgets import QApplication, QDialog, QVBoxLayout, QLabel, QTextEdit, QPushButton, QHBoxLayout, QSizePolicy
+from urllib.parse import quote
+
+FORUM_URI = "https://forum.image.sc/tag/ilastik"
+TEAM_EMAIL = "team@ilastik.org"
+FILE_PATH_MASK = "(file path masked)"
+
+
+def _mask_file_paths(text: str) -> str:
+    """Replace file paths with a placeholder for privacy."""
+    windows_pattern = r"""       # Windows path or file-URI with Windows path (could be with double-backslash)
+        [a-zA-Z]:[/\\]\\?        # Drive and first slash
+        (?!/)                    # Not immediately followed by another slash (avoid matching schema://)
+        ([^/\\\n]+[/\\]\\?)*     # Any number of directories
+        \S*
+    """
+    unix_pattern = r"""          # Unix path or other URI
+            /[^/\n]+/            # at least two slashes, but not double-slash
+            \S*
+    """
+    combined_pattern = re.compile(f"({windows_pattern}|{unix_pattern})", re.X)
+    masked = re.sub(combined_pattern, FILE_PATH_MASK, text)
+    return masked
+
+
+def _get_log_since_last_startup(text_size_limit=65536, pre_marker_padding=512) -> str:
+    """
+    Reads the log up to maximum `text_size_limit` from the end (64 KiB).
+    Returns the text since the last startup plus `pre_marker_padding`, or the entire text if the marker is not found.
+    We don't want people posting or emailing us huge amounts of log...
+    """
+    log_path = ilastik_logging.default_config.get_logfile_path()
+    file_size = int(os.path.getsize(log_path))
+    with open(log_path, "rb") as log_file:
+        position = max(0, file_size - text_size_limit)
+        log_file.seek(position)
+        log_text = log_file.read().decode()
```

Comment:
Because seek/read/etc deals in bytes, this can break if `position` ends up splitting a unicode codepoint:

```python
f = open("/tmp/blas.txt", "w+")
f.write("á")
f.close()

fr = open("/tmp/blas.txt", "rb")
import io
fr.seek(-1, io.SEEK_END)
fr.read().decode()
#UnicodeDecodeError: 'utf-8' codec can't decode byte 0xa1 in position 0: invalid start byte
```

Even if we never write non-ascii in our strings, surely some file paths in the user's machine might have them.